src/modules/pos/validator/components/ValidatorProfile/DetailsView.js

```diff
@@ -44,8 +45,8 @@ const DetailsView = ({ data }) => {
     {
       icon: 'commissionsIcon',
       label: t('Commission'),
-      value: `${extractValidatorCommission(data.commission)}%`,
-      onEdit: onEditCommision,
+      value: `${convertCommissionToPercentage(data.commission)}%`,
+      onEdit: () => addSearchParamsToUrl(history, { modal: 'changeCommission' }),
```

Comment:
@soroushm This icon should only be displayed if the current account is the same as the validator. That way, it prevents the page from breaking

Reply:
good catch, it was part of the other ticket will fix it now